sdk/azidentity/azidentity.go

```diff
@@ -18,19 +18,23 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/errorinfo"
 )
 
+// The base URL for Azure Active Directory
+type AzureAuthorityHost string
```

Comment:
Can we shorten this to `AuthorityHost`?  Having to specify `azidentity.AzureAuthorityHost` seems long and a bit redundant.

Reply:
Sure, have done.